src/ui/components/SpecialDistribution.js

```diff
@@ -41,88 +76,116 @@ export const SpecialDistribution = () => {
       };
       const nextLedger = ledger.distributeGrain(distribution);
       updateLedger(nextLedger);
-      setAmount(fromInteger(0));
-      setRecipient(null);
+      setAmount("");
+      setRecipient(nextLedger.account(recipient.identity.id));
+      setSuccessMessage(
+        `You distributed ${amount} grain to ${recipient.identity.name}!`
```

Comment:
nit: should use the name from currency details, cf: https://github.com/sourcecred/sourcecred/pull/2145